### PR TITLE
tests/hassh Add test cases for hassh

### PR DESCRIPTION
**Pull request by MalakhatkoVadym**
This commit adds test cases for the ssh.hassh, ssh.hassh.string, ssh.hassh.server, ssh.hassh.server.string keywords.

Modifies https://github.com/OISF/suricata-verify/pull/259 by
1. Fixing comments in ssh-banner-only test.

### CODE REVIEW

**Comment by victorjulien** [comment]
Can you also add checks to one of the tests that looks for the various hassh fields and confirms they are correct?

File: tests/ssh-banner-only/test.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+requires:
+  min-version: 5.0.0
```
Comment (victorjulien) [inline tests/ssh-banner-only/test.yaml:2]: these should be 6.0.0

**Comment by catenacyber** [comment]
> Can you also add checks to one of the tests that looks for the various hassh fields and confirms they are correct?

I am not sure I understand what you mean @victorjulien 

Do you mean using alerts as checks against the values is not good ? cf
https://github.com/OISF/suricata-verify/pull/262/files#diff-9dabb85edfb1a42602d72e3013624b99R3

**Comment by MalakhatkoVadym** [comment]
https://github.com/OISF/suricata-verify/pull/263

**Pull request closed without merging**